### DISCUSSION TITLE
How to use it to represent local files and become the kind that can be accessed in the browser

### DISCUSSION THREAD

**Discussion by woyouyitouchenfeijv** [apache/apisix#6472] [Q&A]
We have a lot of static HTML locally and want to use it as a proxy for access

**Comment by tzssangglass** [comment]
ref:https://github.com/apache/apisix/issues/4889

In fact, it is not recommended to use APISIX to proxy static files; APISIX is a dynamic API gateway and is better suited for handling requests.

**Reply by woyouyitouchenfeijv** [reply]
ok 。I found it, but sometimes I can't avoid it. I am currently using nginx agent to register with apisik.
